internal/log/log.go

```diff
@@ -17,17 +17,25 @@
 package log
 
 import (
+	"context"
 	"fmt"
 	"github.com/dynatrace/dynatrace-configuration-as-code/internal/loggers"
 	"github.com/dynatrace/dynatrace-configuration-as-code/internal/loggers/console"
 	"github.com/dynatrace/dynatrace-configuration-as-code/internal/loggers/zap"
 	"github.com/dynatrace/dynatrace-configuration-as-code/internal/timeutils"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/config/v2/coordinate"
```

Comment:
And should we at all assert the type? 
```
	if c, ok := ctx.Value(CtxKeyCoord{}).(coordinate.Coordinate); ok {
```

Why can't we just accept any `interface{}`? If a user passes something wrong, we would not log anything at all, not even a warning that an unexpected type is passed.

Reply:
Moving it another package or dropping the convenience a bit. 
I think it does make sense to assert though, if I stuff in any object accidentally it would be wrong to log it as a coordinate, that might break automation

Reply:
I've left this as is. If we can think of a nice/non-overengineered way to break the dependency in the future while retaining safety that the 'coordinate' field contains a coordinate, let's change it then.